src/Native/pkg/runtime.native.System/runtime.native.System.builds

```diff
@@ -11,10 +11,14 @@
       <OSGroup>debian.8</OSGroup>
       <Platform>amd64</Platform>
     </Project>
-    <Project Include="ubuntu\runtime.native.System.pkgproj">
+    <Project Include="ubuntu\14.04\runtime.native.System.pkgproj">
```

Comment:
You should actually be able to do that already.  OSGroup is sufficient, I'm using this to define runtime in a recent PR: https://github.com/dotnet/corefx/pull/8072.  Could do something similar here.

We should limit the number of things we pass as global properties, it can get out of hand pretty easily and then your left with race conditions and redundant builds.  I think OSGroup and TargetGroup are good for now.